analyzers/src/SonarAnalyzer.CSharp/Rules/DeadStores.RoslynCfg.cs

```diff
@@ -105,6 +108,17 @@ private void ProcessSimpleAssignment(ISimpleAssignmentOperationWrapper assignmen
                 private void ProcessIncrementOrDecrement(IIncrementOrDecrementOperationWrapper incrementOrDecrement) =>
                     ProcessAssignment(incrementOrDecrement, incrementOrDecrement.Target);
 
+                private void ProcessDeconstructionAssignment(IDeconstructionAssignmentOperationWrapper deconstructionAssignment)
+                {
+                    if (ITupleOperationWrapper.IsInstance(deconstructionAssignment.Target))
+                    {
+                        foreach (var tupleElement in ITupleOperationWrapper.FromOperation(deconstructionAssignment.Target).AllElements())
+                        {
+                            ProcessAssignment(deconstructionAssignment, tupleElement);
```

Comment:
[education] why is this different from `ProcessSimpleAssignment` which looks at the result of `ProcessAssignment`?

Reply:
You are right. I added a test case and fixed the issue